token/shared/shared.go

```diff
@@ -96,16 +95,20 @@ func convertTo29BitsAndHalf(source uint64) uint32 {
 	return uint32(temp)
 }
 
-func convertTo4DigitsToken(source uint64) string {
+func ConvertTo4DigitsToken(source uint64) string {
 	var restrictedDigitToken strings.Builder
 
-	bitArray := getBitArrayFromInt(source, 30)
+	bitArray := getBitArrayFromInt(source, 4*2)
 
-	for i := range 15 {
-		thisArray := bitArray[i*2 : (i*2)+2]
-		restrictedDigitToken.WriteString(
-			fmt.Sprint(bitArrayToInt(thisArray) + 1))
-	}
+	//for i := range 4 {
+	//	thisArray := bitArray[i*2 : (i*2)+2]
+	//	restrictedDigitToken.WriteString(
+	//		fmt.Sprint(bitArrayToInt(thisArray) + 1))
+	//}
```

Comment:
Is this commented out only temporary? Or could we remove it?

Reply:
I commented this part out as it didn't give the expected result from the tests. It turns out ```ir := bitArrayToInt(bitArray)``` was enough to get the function to work correctly. 

I will remove it if no further comments on its usefulness is added.

Reply:
Updated.